src/components/network-map-tab.js

```diff
@@ -658,12 +661,14 @@ export const NetworkMapTab = ({
             ) {
                 loadMissingGeoData();
             } else {
-                loadAllGeoData();
+                loadRootNodeGeoData();
+                // set initialized to false to trigger missing geo-data fetching on next render
+                setInitialized(false);
             }
         }
-    }, [studyUuid, loadAllGeoData, loadMissingGeoData, lineFullPath]);
+    }, [studyUuid, loadRootNodeGeoData, loadMissingGeoData, lineFullPath]);
 
-    const loadMapEquipments = useCallback(() => {
+    const loadDefaultNodeMapEquipments = useCallback(() => {
```

Comment:
Should not be rename nope ?
This method loads the equipments of the current node